common/const.go

```diff
@@ -33,50 +30,16 @@ var (
 	// GlobalURL URL for fetching the global config.
 	GlobalURL = "https://globalconfig.flashlightproxy.com/global.yaml.gz"
 
-	// GlobalStagingURL is the URL for fetching the global config in a staging environment.
-	GlobalStagingURL = "https://globalconfig.flashlightproxy.com/global.yaml.gz"
-
-	// StagingMode if true, run Lantern against our staging infrastructure.
-	// This is set by the linker using -ldflags
-	StagingMode = "false"
-
-	Staging = false
-
 	ProAPIHost = "api.getiantem.org"
 
 	log = golog.LoggerFor("flashlight.common")
 
-	forceAds bool
-
 	// Set by the linker using -ldflags in the project's Makefile.
 	// Defaults to 'production' so as not to mistakingly push development work
 	// to a production environment
 	Environment = "production"
 )
-
-func init() {
-	initInternal()
-}
-
-// ForceStaging forces staging mode.
-func ForceStaging() {
-	StagingMode = "true"
-	initInternal()
-}
-
-func initInternal() {
-	var err error
-	log.Debugf("****************************** stagingMode: %v", StagingMode)
-	Staging, err = strconv.ParseBool(StagingMode)
-	if err != nil {
-		log.Errorf("Error parsing boolean flag: %v", err)
-		return
-	}
-	if Staging {
-		ProAPIHost = "api-staging.getiantem.org"
-	}
-	forceAds, _ = strconv.ParseBool(os.Getenv("FORCEADS"))
-}
+var forceAds, _ = strconv.ParseBool(os.Getenv("FORCEADS"))
```

Comment:
I guess this can also be removed with the function below, I don't see any reference or usage (it looks like the same file exists in lantern-client `internalsdk/common/const.go` and also is not being used)

Reply:
probably this whole package shouldn't exist, this definitions are only used at specific packages and most stuff is unused

Reply:
Agreed forceads can go